modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -36,7 +56,7 @@ $ NAMESPACE=nginx-lb-test
 $ oc create ns $NAMESPACE
 ----
 
-. The following example deploys three replicas of the test `nginx` application in your namespace:
+. The following example deploys three replicas of the test `nginx` application in the created namespace:
```

Comment:
This is out of scope to your changes, but just an FYI that [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps) the first sentence of a step should use the imperative mood, and this step does not seem to have any explicit instruction for users in it.